### PR TITLE
feat(strm-1593): allow arbitrary annotations on the load balancer

### PR DESCRIPTION
**Pull request by trietsch**
Arbitrary annotations allow more control for the end user installing the Helm chart.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace:
 	kubectl create namespace ${namespace}
 install:
 	-helm status ${helm_name} --namespace  ${namespace} 2> /dev/null \
-	|| helm install ${helm_name} helm --namespace ${namespace} \
+	|| helm install --generate-name helm --namespace ${namespace} \
```
Comment (bvdeenen) [inline Makefile:17]: This will create a random name afaik. Why the change?

**Review by bvdeenen — APPROVED** [review]
Why the change to a generated name? Has this been reflected in the documentation

File: Makefile (hunk 1 above)
Comment (trietsch) [reply Makefile:17]: This is by accident, I'll change this.
